stable/keycloak/values.yaml

```diff
@@ -49,7 +49,10 @@ keycloak:
   password: ""
 
   ## Allows the specification of additional environment variables for Keycloak
+  ## Set PROXY_ADDRESS_FORWARDING if using ingress
```

Comment:
This is already enabled and the env var is useless. Should we change this such that one has to set the env var?

See https://github.com/helm/charts/blob/master/stable/keycloak/values.yaml#L128

Reply:
Yes I think it is redundant as it is set using the jboss cli part of the values you mentioned so I will remove it.  After doing more testing i didn't need to set the PROXY_ADDRESS_FORWARDING anymore.